src/py_strapi/export/importer.py

```
@@ -0,0 +1,489 @@
+                # Track ID mapping
+                media_id_mapping[exported_media.id] = uploaded.id
+                result.media_imported += 1
+
+            except Exception as e:
+                result.add_warning(f"Failed to import media {exported_media.name}: {e}")
+                result.media_skipped += 1
+
+        logger.info(f"Imported {result.media_imported}/{len(export_data.media)} media files")
+        return media_id_mapping
+
+    def _load_schemas_from_export(self, export_data: ExportData, result: ImportResult) -> None:
```

Ruff (0.14.14)

[warning] Unused method argument: `result`

(ARG002)